NEWS.md

```diff
@@ -1,4 +1,18 @@
-# cchsflow 1.2.0 (Latest build)
+# cchsflow 1.2.1 (Latest build)
+2020-01-21
```

Comment:
remember to change date when we submit :')

Reply:
Also, add to the note the other changes and additions. 
changes to message
addition of food insecurity

Reply:
scratch that. I see those notes now.